lib/win32/windows/time_calc_helper.rb

```diff
@@ -0,0 +1,116 @@
+module Windows
+  module TimeCalcHelper
+
+    DAY_OF_MONTH = [0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
+
+    # Returns no of days in a given month of a year
+    def day_in_month(month, year)
+      (month == 2 && is_leap_year?(year)) ? 29 : DAY_OF_MONTH[month]
+    end
+
+    # Year is leap when it is a multiple of 4 and not a multiple of 100.
+    # But it can be a multiple of 400
+    def is_leap_year?(year)
+      (((year % 4).zero? && !(year % 100).zero?) || (year % 400).zero?)
+    end
+
+    # Returns total time duration in minutes
+    def time_in_minutes(time_str)
+      time_in_seconds(time_str) / 60
+    end
+
+    # Calculates total time duration in seconds
+    def time_in_seconds(time_str)
+      dt_tm_hash = time_details(time_str)
+      curr_time = Time.now
+
+      # Basic time variables
+      future_year = curr_time.year + dt_tm_hash[:year].to_i
+      future_mth = curr_time.month + dt_tm_hash[:month].to_i
```

Comment:
can we call this `future_month` instead of `future_mth` since we use month in full everywhere else?

Reply:
Have updated the variable name. Thank you